src/admin/config.njk

```diff
@@ -0,0 +1,70 @@
+---
+permalink: /admin/config.yml
+---
+backend:
+  name: git-gateway
+  branch: main
+media_folder: "src/assets/uploads"
+public_folder: "/assets/uploads"
+
+# Run "npx netlify-cms-proxy-server" for local backend
+local_backend: true
+editor:
+  preview: false
+i18n:
+  structure: multiple_folders
+  locales: [{% for code, language in config.languages %}{{code}}{% if loop.last !== true %}, {% endif %}{% endfor %}]
```

Comment:
Is the `language` variable needed here?

Reply:
Yes— this is the only downside to the change in `config.languages` structure. When [iterating over an object](https://mozilla.github.io/nunjucks/templating.html#for), Nunjucks requires the following syntax:

```njk
{% for key, value in object %}
{# Do something… #}
{% endfor %}
```

All that's needed here is the key but I'm pretty sure I can't do this:

```njk
{% for key in Object.keys(object) %}
{# Do something… #}
{% endfor %}
```

So the `language` variable is needed.

Reply:
Thanks for the link to the nunjucks docs. Probably not worth changing to the second format even if it is possible.